src/ofxGLWarper.cpp

```diff
@@ -12,8 +17,6 @@ void ofxGLWarper::setup(int _resX, int _resY){
 //--------------------------------------------------------------
 void ofxGLWarper::setup(int _x, int _y, int _w, int _h){
     ofLogVerbose() << "ofxGLWarper setup: " <<_x << " " <<_y << " " <<_w << " " <<_h << endl;
-    ofUnregisterMouseEvents(this);
```

Comment:
I am not sure if this is safe. Why would you want to have this?

Reply:
I see. You mean if it was already active (setup twice). I added a deactivate call that will work if necessary.